src/app/tournament/tournament-routing.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { TournamentComponent } from './tournament/tournament.component';
+import { CsmBracketComponent } from './bracket/csm-bracket/csm-bracket.component';
+import { TtuBracketComponent } from './bracket/ttu-bracket/ttu-bracket.component';
+import { GrandFinalComponent } from './bracket/grand-final/grand-final.component';
+
+const routes: Routes = [
+  {path: '', component: TournamentComponent, children: [
+    { path:'', redirectTo: 'FINALS', pathMatch: 'full' },
+    { path: 'FINALS', component: GrandFinalComponent},
```

Comment:
Each of these child routes, with the exception of the key, look to be the same component.  Is there a reason we are making them different components?

Reply:
Good call, I implemented the data pass through the routes instead of building new components